test/re/frida/ClassCreationTest.java

```diff
@@ -25,6 +25,7 @@
     private static Class simplePrimitiveArrayClass = null;
     private static Class myOutputClass1 = null;
     private static Class myOutputClass2 = null;
+    private static Class userDefinedFieldClass = null;
```

Comment:
Let's move this after `orangeClass`, so these are defined in the same order as they're used.